nexus/src/app/crucible.rs

```diff
@@ -146,12 +147,14 @@ impl super::Nexus {
         }
     }
 
-    /// Call out to Crucible agent and perform region creation.
-    async fn ensure_region_in_dataset(
+    /// Call out to Crucible agent and perform region creation. Optionally,
+    /// supply a read-only source to invoke a clone.
+    pub async fn ensure_region_in_dataset(
         &self,
         log: &Logger,
         dataset: &db::model::Dataset,
         region: &db::model::Region,
+        source: Option<String>,
```

Comment:
The changes in this file are all format related (I think) except for this one.
This change is not required for this PR (but will be soon) correct?

Reply:
That's right yeah.